docs/includes/file-get-started-excel-vue.md

```diff
@@ -52,28 +54,27 @@ Each add-in requires a manifest file to define its settings and capabilities.
     - **What do you want to name your add-in?:** `My Office Add-in`
     - **Which Office client application would you like to support?:** `Excel`
 
-    After you complete the wizard, a manifest file and resource file are available for you to build your project.
-
     ![Yeoman generator](../images/yo-office.png)
     
-    > [!NOTE]
-    > If you're prompted to overwrite **package.json**, answer **No** (do not overwrite).
+    After you complete the wizard, the generator will create the manifest file.
 
 ## Secure the app
 
 [!include[HTTPS guidance](../includes/https-guidance.md)]
 
-To enable HTTPS for your app, open **package.json** in the root of the project, modify the `dev` script to add the `--https` flag, and save the file.
+To enable HTTPS for your app, open the file **package.json** in the root folder of the Vue project, modify the `dev` script to add the `--https` flag, and save the file.
 
 ```json
 "dev": "webpack-dev-server --https --inline --progress --config build/webpack.dev.conf.js"
 ```
 
 ## Update the app
 
-1. In your code editor, open the manifest file (i.e., the file in the root directory of your app with a name ending in "manifest.xml"). Replace all occurrences of `https://localhost:3000` with `https://localhost:8080` and save the file.
+1. In your code editor, open the folder **My Office Add-in** that Yo Office created at the root of your Vue project. In that folder, you'll see the manifest file that defines the settings for your add-in: **my-office-add-in-manifest.xml**.
+
+2. Open the manifest file, replace all occurrences of `https://localhost:3000` with `https://localhost:8080`, and save the file.
```

Comment:
General question: why is this step necessary? Are we trying to show how to change where the add-in looks for the service? Or is this a Vue specific step?

Reply:
It's Vue-specific -- by default, a locally-hosted Vue app runs on port 8080.